website/package.json

```diff
@@ -21,11 +21,12 @@
     "@uiw/react-github-corners": "^1.5.16",
     "d3-regression": "^1.3.10",
     "date-fns": "^4.1.0",
+    "framer-motion": "^12.15.0",
     "fusioncharts": "4.1.2",
     "highcharts-react-official": "^3.2.2",
     "papaparse": "^5.5.3",
-    "react": "^19.1.0",
-    "react-dom": "^19.1.0",
+    "react": "^18.3.1",
```

Comment:
Ideally it would be nice to use latest react version, I think the problem you might have seen is `fusionchart` not being compatible with latest React. What I'm now doing is to run `npm i --force` to keep using React 19.